src/Cardano/Wallet/Binary.hs

```diff
@@ -600,3 +621,9 @@ decodeListIndef decodeOne = do
 blake2b256 :: forall tag. CBOR.Encoding -> Hash tag
 blake2b256 =
     Hash . BA.convert . hash @_ @Blake2b_256 . CBOR.toStrictByteString
+
+
+encodeList :: (a -> CBOR.Encoding) -> [a] -> CBOR.Encoding
+encodeList encodeOne l = mempty
+    <> CBOR.encodeListLen (fromIntegral $ length l)
+    <> foldl (\a b -> (a <> encodeOne b)) mempty l
```

Comment:
since everything is a monoid here, you can also just `mconcat` a list of `Encoding`. `mconcat (encodeOne <$> l)`

Reply:
Thanks! Fixed.